ravenframework/Optimizers/GradientDescent.py

```diff
@@ -126,9 +126,9 @@ def getInputSpecification(cls):
               it can be accepted. Exactly one of the acceptance criteria
               below may be selected for this Optimizer.""")
     specs.addSub(accept)
-    ## common options to all acceptanceCondition descenders
-    ## TODO
-    ## get specs for each acceptanceCondition subclass, and add them to this class's options
+    # common options to all acceptanceCondition descenders
+    # TODO
+    # get specs for each acceptanceCondition subclass, and add them to this class's options
```

Comment:
I think we can remove this TODO, as the next couple lines appear to do just that.

Reply:
Okay, I'll remove it.